File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_04659.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest04659():
    xml_value = request.get_data(as_text=True)
    data = xml_value if xml_value else 'default'
    return jsonify({'status': 'ok'}), 200, {'Content-Language': str(data)}

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest04659.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest04659 {

    @GetMapping("/BenchmarkTest04659")
    public void BenchmarkTest04659(@RequestParam("id") String id, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String userId = id != null ? id : "";
        java.util.function.Function<String, String> initialFn = s -> s.replaceAll("[\\u0000-\\u001F]", "");
        java.util.function.Function<String, String> transformed = initialFn.andThen(String::strip);
        String data = transformed.apply(userId);
        String processed = data.length() > 64 ? data.substring(0, 64) : data;
        byte[] buf = new byte[Integer.parseInt(processed)];
        response.setContentType("application/json");
        response.getWriter().print("{\"id\":0}");
    }
}
